src/main/kotlin/com/goodchoice/infra/config/SecurityConfig.kt

```diff
@@ -36,6 +35,14 @@ class SecurityConfig(private val authService: AuthService) : WebSecurityConfigur
                 authorize(GET, "/docs/swagger-config", permitAll)
 
                 authorize(POST, "/reviewers", permitAll)
+
+                // Brands
+                authorize(GET, "/brands", permitAll)
+                authorize(GET, "/brands/*", permitAll)
+                authorize(PUT, "/brands", authenticated)
+                authorize(POST, "/brands", authenticated)
```

Comment:
We can specify URLs for `permitAll` only. All other cases will be matched by last rule `authorize("/**", authenticated)`